scripts/sensorshow

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python3
+
+'''
+    Script to show Voltage and Current Sensor status.
+'''
+from tabulate import tabulate
+from swsscommon.swsscommon import SonicV2Connector
+from natsort import natsorted
+import argparse
+
+
+header = ['Sensor', 'Voltage(mV)', 'High TH', 'Low TH', 'Crit High TH', 'Crit Low TH', 'Warning', 'Timestamp']
```

Comment:
Since there is a get_unit function, maybe we should not make assumption that the unit is "mV"